File: fxj/project/fxj/vue-admin-template-master/src/api/module/memory/record.js
import request from '@/utils/request'

/**
 * 得到record列表
 */
export function getRecordList(id, anotherId) {
  return request({
    url: `/memory/record/${id}/${anotherId}`,
    method: 'get'
  })
}

/**
 * 条件查询record列表
 */
export function getRecordListByQuery(id, anotherId, searchObj) {
  return request({
    url: `/memory/record/query/${id}/${anotherId}`,
    method: 'post',
    data: searchObj
  })
}

export function updateRecord(record) {
  return request({
    url: `/memory/record`,
    method: 'post',
    data: {
      id: record.id,
      recordId: record.recordId,
      date: record.date,
      num: record.num,
      content: record.content,
      type: record.type,
      tag: record.tag,
      isDeleted: record.isDeleted
    }
  })
}

File: src/main/java/com/myweb/fxj/pojo/memory/Record.java
package com.myweb.fxj.pojo.memory;

import com.baomidou.mybatisplus.annotation.TableName;
import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;
import org.springframework.format.annotation.DateTimeFormat;

import java.util.Date;

@Data
@TableName("memory_record")
public class Record {
    private String type; // 记录类型
    private Integer recordId; // 记录id
    private Integer id; // 记录属于的id
    @DateTimeFormat(pattern = "yyyy-MM-dd")
    @JsonFormat(pattern = "yyyy-MM-dd", locale = "zh", timezone = "GMT+8")
    private Date date; // 记录日期
    private String content; // 记录内容
    private String tag;
    private Integer isDeleted;
}
